src/plugins/lights/public/js/lights.js

```diff
@@ -32,47 +32,48 @@
             }
 
             // Setup input handlers
-            this.inputDefaults = 
-            [
-              {
+            this.inputDefaults = [
+            {
                 name: 'plugin.lights.stepPositive',
                 description: 'Makes the ROV lights brighter.',
-                defaults: 
-                {
-                  keyboard: 'p',
-                  gamepad: 'DPAD_UP'
-                },
-                down: function () 
+                shortName: "Increase ROV Light Brightness",
```

Comment:
Is the shortName concept really doing what it is intended to do. I don't see any apparent benefit over the description.